CHANGELOG.md

```diff
@@ -34,12 +37,61 @@ All notable changes to this project will be documented in this file.
 - `1.0.x` Releases - [1.0.0](#100) | [1.0.1](#101)
 
 ---
+## [5.0.0.beta.1](https://github.com/Alamofire/Alamofire/releases/tag/5.0.0.beta.1)
+Released on 2018-12-06. All issues associated with this milestone can be found using this [filter](https://github.com/Alamofire/Alamofire/issues?utf8=✓&q=milestone%3A5.0.0.beta.1). **Note:** Alamofire will not be following semantic versioning during the beta process. There may be breaking changes until 5.0.0 is released.
```

Comment:
Instead of saying until `5.0.0 is released`, maybe something more like until we move into the `rc` process for 5.0.0 or something like that.

Reply:
I mainly wanted to keep our options open, even if we wanted to do RCs. If you want stable RCs, I can make the suggested change.

Reply:
I think that's fair if you'd like to keep the `rc` option open.